### PR TITLE
Remove some integration tests from deprecated drivers and switch more…

### PR DESCRIPTION
**Pull request by dlorenc**
… to kubeadm.

cc @r2d4

### CODE REVIEW

File: hack/jenkins/linux_integration_tests_kvm.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@
 set -e
```
Comment (r2d4) [inline hack/jenkins/linux_integration_tests_kvm.sh]: You should remove the entries from the `set-pending` script also

Comment (dlorenc) [reply hack/jenkins/linux_integration_tests_kvm.sh]: Good catch. Done.

File: hack/jenkins/linux_integration_tests_none.sh [hunk 2]
```diff
@@ -29,7 +29,7 @@ set -e
 OS_ARCH="linux-amd64"
 VM_DRIVER="none"
 JOB_NAME="Linux-None"
-EXTRA_ARGS="--bootstrapper=localkube"
+EXTRA_ARGS="--bootstrapper=kubeadm"
 EXTRA_START_ARGS="--kubernetes-version=file://$PWD/out/localkube"
```
Comment (r2d4) [inline hack/jenkins/linux_integration_tests_none.sh]: remove the --kubernetes-version flag

File: hack/jenkins/osx_integration_tests_hyperkit.sh [hunk 3]
```diff
@@ -29,7 +29,7 @@ set -e
 OS_ARCH="darwin-amd64"
 VM_DRIVER="hyperkit"
 JOB_NAME="OSX-Hyperkit"
-EXTRA_ARGS="--bootstrapper=localkube"
+EXTRA_ARGS="--bootstrapper=kubeadm"
 EXTRA_START_ARGS="--kubernetes-version=file://$PWD/out/localkube"
```
Comment (r2d4) [inline hack/jenkins/osx_integration_tests_hyperkit.sh]: remove the --kubernetes-version flag

**Pull request merged**